cmd/launcher/svc_windows.go

```diff
@@ -232,3 +244,74 @@ func (w *winSvc) Execute(args []string, r <-chan svc.ChangeRequest, changes chan
 		}
 	}
 }
+
+// determineRootDirectory is used specifically for windows deployments to override the
+// configured root directory if another one containing a launcher DB already exists
+func determineRootDirectory(optsRootDirectory string, slogger *slog.Logger) string {
+	optsDBLocation := filepath.Join(optsRootDirectory, "launcher.db")
+	// database already exists in configured root directory, keep that
+	dbExists, err := nonEmptyFileExists(optsDBLocation)
```

Comment:
I think there's another case here.... If the host _isn't_ the prod URL, then we should probably take whatever specified rootdirectory is, because it's probably dev

Reply:
thank you, pushed up a change- let me know if that's what you're looking for